starters/blog/src/components/bio.js

```diff
@@ -36,30 +36,34 @@ const Bio = () => {
   `)
 
   const { author, social } = data.site.siteMetadata
+  const avatar = data.avatar?.childImageSharp?.fixed
+
   return (
     <div
       style={{
         display: `flex`,
         marginBottom: rhythm(2.5),
       }}
     >
-      <Image
-        fixed={data.avatar.childImageSharp.fixed}
-        alt={author.name}
-        style={{
-          marginRight: rhythm(1 / 2),
-          marginBottom: 0,
-          minWidth: 50,
-          borderRadius: `100%`,
-        }}
-        imgStyle={{
-          borderRadius: `50%`,
-        }}
-      />
+      {avatar && (
+        <Image
+          fixed={avatar}
+          alt={author?.name || ``}
+          style={{
+            marginRight: rhythm(1 / 2),
+            marginBottom: 0,
+            minWidth: 50,
+            borderRadius: `100%`,
+          }}
+          imgStyle={{
+            borderRadius: `50%`,
+          }}
+        />
+      )}
       <p>
-        Written by <strong>{author.name}</strong> {author.summary}
+        Written by <strong>{author?.name || `Author`}</strong> {author?.summary || `Author Summary`}
```

Comment:
How about letting them remove the summary, and conditionally showing this `p` according to whether name is set